### PR TITLE
Top notch clipping in XS and XR fix

### PR DESCRIPTION
**Pull request by imashok02**

### CODE REVIEW

File: tealeaf/platform/TeaLeafViewController.mm [hunk 1]
```diff
@@ -255,12 +255,22 @@ - (void) destroyGLView {
...
     
+	glView.translatesAutoresizingMaskIntoConstraints = NO;
+    
+	if (@available(iOS 11.0, *)) {
+		UILayoutGuide * layoutGuide = self.view.safeAreaLayoutGuide;
+		[glView.leadingAnchor constraintEqualToAnchor:layoutGuide.leadingAnchor].active = YES;
+		[glView.trailingAnchor constraintEqualToAnchor:layoutGuide.trailingAnchor].active = YES;
+		[glView.topAnchor constraintEqualToAnchor:layoutGuide.topAnchor].active = YES;
+		[glView.bottomAnchor constraintEqualToAnchor:layoutGuide.bottomAnchor].active = YES;
+	}
```
Comment (rampr) [inline tealeaf/platform/TeaLeafViewController.mm:272]: will this work for iphone x ?

Comment (imashok02) [reply tealeaf/platform/TeaLeafViewController.mm:272]: yes @rampr, this will work.

Comment (rampr) [reply tealeaf/platform/TeaLeafViewController.mm:272]: can we remove the old change we made for iphone x then ?

Comment (imashok02) [reply tealeaf/platform/TeaLeafViewController.mm:272]: I was thinking about it, will do the same

**Review by vivekb91 — APPROVED** [review]

File: tealeaf/platform/TeaLeafViewController.mm (hunk 1 above)
Comment (imashok02) [reply tealeaf/platform/TeaLeafViewController.mm:272]: old change also removed.

**Review by rampr — APPROVED** [review]

**Pull request merged**